build.gradle

```diff
@@ -300,63 +301,55 @@ war {
 
 assemble.dependsOn copyDeps
```

Comment:
Do you want to leave this as jar.dependsOn. 

For example, we run ./gradlew clean tar, which produces the tar ball of voldemort and we send the tar ball to other machines for running the command line and testing. now build and tar will probably work, but that is a change in behavior.